src/test/java/com/ing/data/cassandra/jdbc/DataSourceUnitTest.java

```diff
@@ -20,7 +20,7 @@
 import java.sql.SQLException;
 import java.util.Collections;
 
-import static com.ing.data.cassandra.jdbc.utils.JdbcUrlUtil.TAG_CONSISTENCY_LEVEL;
+import static com.ing.data.cassandra.jdbc.utils.JdbcUrlUtil.*;
```

Comment:
Be careful to not have `*` imports and only import the necessary classes or constants.